src/installer/installer.go

```diff
@@ -453,6 +453,22 @@ func (i *installer) startBootstrap() error {
 		return err
 	}
 
+	// If we're in a pure RHEL/CentOS environment, we need to overlay the node image
+	// first to have access to e.g. oc, kubelet, cri-o, etc...
+	// https://github.com/openshift/enhancements/pull/1637
+	if !i.ops.FileExists("/usr/bin/oc") {
+		err = i.ops.SystemctlAction("start", "node-image-overlay.service")
+		if err != nil {
+			return err
+		}
+
+		i.waitForNodeImageOverlay(context.Background())
+
+		if !i.ops.FileExists("/usr/bin/oc") {
+			return stderrors.New("/usr/bin/oc still doesn't exist after node image overlay")
```

Comment:
Is this the only failure mode? Is there any way this does exist, but something else didn't get extracted properly?

I'd be more comfortable if we could check the actual status of the services to ensure nothing failed.